### ISSUE TITLE
Trying `sampled_softmax_loss`

### ISSUE THREAD

**Issue by bkj** [davidsandberg/facenet#350]
Has anyone ever tried to use `tf.nn.sampled_softmax_loss` instead of `tf.nn.sparse_softmax_cross_entropy_with_logits` in `train_softmax.py`?  

Sampled softmax loss can give a big speedup when the number of classes are very large.  Not sure exactly what the speedup would be for the CASIA dataset (10K classes), but probably nontrivial.

**Comment by davidsandberg** [comment]
Closing this for now. Reopen if needed.
